android/build.gradle

```diff
@@ -30,5 +30,5 @@ repositories {
 }
 
 dependencies {
-    compile 'com.facebook.react:react-native:0.15.+'
+    compile 'com.facebook.react:react-native:+'
```

Comment:
We might need a minimum dependency here as it seems there were breaking changes at some point.